core/src/test/java/org/infinispan/eviction/impl/MemoryBasedEvictionFunctionalStoreAsBinaryTest.java

```diff
@@ -39,8 +37,4 @@ public void testCustomClass() throws Exception {
       }
       assertTrue(cache.getAdvancedCache().getDataContainer().size() < numberInserted);
    }
-
-   public void testJGroupsAddress() {
-      cache.put("key", new JGroupsAddress(new UUID()));
-   }
```

Comment:
Why the removal?

Reply:
This test no longer makes sense with the separation of the internal and user contexts, as it's trying to store the internal class `JGroupsAddress`.

We could keep the test and add a `JGroupsAddress` SCI to the configuration, but I don't think this is worthwhile as `testCustomClass` essentially does the same thing albeit with a different class.